doc/schema.dot

```diff
@@ -0,0 +1,72 @@
+// dot schema.dot -Tpng -o schema.png -Gdpi=100
+digraph G {
+    rankdir="LR";
+    overlap=scalexy;
+    node [shape=box
+          style="rounded,filled"]
+
+    webapp[label=<
+           <table border="0">
+            <tr>
+             <td><img src="icon_tornado.png"/></td>
+            </tr>
+            <tr>
+             <td>webapp</td>
+            </tr>
+           </table>>
+           fillcolor=lightblue]
+
+    websocket[label=<
+           <table border="0">
+            <tr>
+             <td><img src="icon_tornado.png"/></td>
+            </tr>
+            <tr>
+             <td>websocket</td>
+            </tr>
+           </table>>
+           fillcolor=lightblue]
+
+    reposcan[label=<
+           <table border="0">
+            <tr>
+             <td><img src="icon_tornado.png"/></td>
+            </tr>
+            <tr>
+             <td>reposcan</td>
+            </tr>
+           </table>>
+           fillcolor=lightblue]
+
+    db[label=<
+           <table border="0">
+            <tr>
+             <td><img src="icon_postgresql.png"/></td>
+            </tr>
+            <tr>
+             <td>database</td>
+            </tr>
+           </table>>
+           fillcolor=orange
+           shape=cylinder]
+
+    client [fillcolor="pink"]
+
+    clientp [label="client\n(privileged)"
+             fillcolor="pink"]
+
+    client -> webapp [label="http:8080"]
+    clientp -> reposcan [label="http:8081"]
+
+    subgraph cluster_os {
+        margin=25
+        label=OpenShift
```

Comment:
I'd call it like "OpenShift/docker-compose", it's not deployable only in OpenShift.